x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -416,13 +410,53 @@ const updateExistingIndex = async ({
     // if update fails, rollover data stream
   } catch (err) {
     try {
+      // get the data_stream values to compose datastream name
+      const searchDataStreamFieldsResponse = await callCluster('search', {
```

Comment:
nit: I think we could limit the `_source` so only the three fields we care about are returned: https://www.elastic.co/guide/en/elasticsearch/reference/7.9/search-fields.html#source-filtering